scripts/run_tests_unit.sh

```diff
@@ -19,5 +19,6 @@ function display_result {
   fi
 }
 
-pipenv run py.test "$@"
+py.test --cov-config=.coveragerc --cov --cov-report html
```

Comment:
Given there are 100+ unit tests now, we could use https://pypi.org/project/pytest-xdist/ like we do with runner but only a nice to have.